src/init.cpp

```diff
@@ -1464,10 +1460,18 @@ bool AppInitMain(InitInterfaces& interfaces)
             bool is_coinsview_empty;
             try {
                 LOCK(cs_main);
+                g_chainstate.reset(new CChainState(
```

Comment:
In commit "refactor: have CCoins* data managed under CChainState" (eed4e62ad6b162ff0d3e2b87eb8762a689678bca)

Suggest `x = MakeUnique<X>()` instead of `x.reset(new X())` to avoid raw pointers and new in C++, which are suspicious.